eo-runtime/src/main/java/org/eolang/Main.java

```diff
@@ -172,6 +171,9 @@ private static boolean parse(final String opt) throws IOException {
      * Run this opts.
      * @param opts The opts left
      * @throws Exception If fails
+     * @todo #2591:60min Make type casting when logging an app dataization result. The cast is
+     *  necessary in this expression {@code new Data set(app).take()} in order to correctly
```

Comment:
@c71n93 I didn't really get what the expression in the {@code ...} means

Reply:
It's a code snippet.

Reply:
@c71n93 I see but what `new Data set(app)` means?

Reply:
@maxonfjvipon It is an expression, that needs to be casted to an appropriate type.

Reply:
@c71n93 is it a valid Java code?

Reply:
@c71n93 maybe you meant Dataized instead of Data set?

Reply:
@maxonfjvipon Oh, yeah, didn't notice the mistake, thank you

Reply:
@maxonfjvipon done